minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -443,16 +422,9 @@ public void run() {
   protected abstract LocalAlluxioMaster getMaster();
 
   /**
-   * @return the master's {@link Configuration}
-   */
-  public Configuration getMasterConf() {
-    return mMasterConf;
-  }
-
-  /**
    * Resets contexts. This is called when the cluster is stopped.
    */
-  protected void resetContext() {}
+  protected abstract void resetContext();
```

Comment:
could the implementation be here? I think the subclasses implement this identically